tests/kernel/mbox/mbox_api/src/test_mbox_api.c

```diff
@@ -564,6 +563,73 @@ void test_mbox_kdefine(void)
 	tmbox(&kmbox);
 }
 
+/**
+ *
+ * @brief Test mailbox enhance capabilities
+ *
+ * @details
+ * - Define and initilized a message queue and a mailbox
+ * - Verify the capability of message queue and mailbox
+ * - with same data.
+ *
+ */
+void test_enhance_capability(void)
+{
+	int MSG = 0xABCD;
+	struct k_msgq msgq;
+
+	ZTEST_BMEM char __aligned(4) buffer[8];
+	k_msgq_init(&msgq, buffer, 4, 2);
+	/* send buffer with message queue */
+	int ret = k_msgq_put(&msgq, &MSG, K_NO_WAIT);
+
+	zassert_false(ret, "message queue put successful");
```

Comment:
Nitpick: `zassert_false(ret)` to detect ret == 0, which is a **successfull** result, is horrifyingly confusing.  Use `zassert_true(ret == 0)` or `zassert_equal(ret, 0)`, etc...

Reply:
OK, I will fixing it.

Reply:
I have fixed it.